modules/CreditsPage/components/Credit/Credit.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import Link from 'next/link';
+import type { ICredit } from './ICredit';
+import GithubIcon from '../../../shared/components/icons/github.svg';
+import LinkedinIcon from '../../../shared/components/icons/linkedin.svg';
+import DeticatedRank from '../../../shared/components/icons/deticatedRank.svg';
+import Box from '../../../shared/components/atoms/Box/Box';
+import ImgWithInfo from '../../../shared/components/molecules/ImgWithInfo/ImgWithInfo';
+
+const Credit: FC<{ creditData: ICredit.ICreditData }> = ({
+  creditData,
+}): ReactElement => (
+  <Box isWhiteColor classes="max-w-29xl">
+    <>
+      <Box.Header>
+        <>
+          <ImgWithInfo>
+            <>
+              <ImgWithInfo.Image
+                ImageSize="super-large"
+                ImageVariant="avatar"
+                profilePic={creditData.developerImage}
+              />
+              <ImgWithInfo.Info classes="h-full">
+                <>
+                  <ImgWithInfo.Info.Title titleSize="medium">
+                    {creditData.developerName}
+                  </ImgWithInfo.Info.Title>
+                  <ImgWithInfo.Info.SubTitle subTitleSize="small">
+                    {creditData.developerTitle}
+                  </ImgWithInfo.Info.SubTitle>
+                  <ImgWithInfo.Info.MoreInfo>
+                    <ul className="list-none flex">
```

Comment:
clicking on links should navigate the user to a new tab and keep Pickify's tab active